src/lib/components/chat/Messages/UserMessage.svelte

```diff
@@ -20,18 +20,17 @@
 
 	let edit = false;
 	let editedContent = '';
-
+	let messageEditTextArea: HTMLTextAreaElement;
```

Comment:
We might want to name it `messageEditTextAreaElement` to denote that it's an element variable!

Reply:
i missed that one, fixing now